CMakeLists.txt

```diff
@@ -59,6 +59,26 @@ if(NOT DEFINED VCPKG_BUILD_DEFAULT)
   set(VCPKG_TARGET_TRIPLET "${VCPKG_HOST_ARCH}-${VCPKG_HOST_OS}-release")
 endif()
 
+set(VCPKG_DOWNLOADS "/bad/directory/for/vcpkg/downloads")
```

Comment:
Hmm, curious. What's the reason for this?

Reply:
Bogus, I'll fix it.